acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/timeouts/TimeoutAcceptanceTest.java

```diff
@@ -43,9 +44,10 @@
   @Before
   public void setUp() {
     final NodeConfiguration nodeConfig = new NodeConfigurationBuilder().build();
+    final NodePorts nodePorts = new NodePorts(8545, 8456);
```

Comment:
There is a nice constant defined in JsonRpcConfiguration. Can this be used here? And I noticed that this number is used a LOT and I do not like these "magic numbers" very much...

Reply:
I'm not seeing that class, perhaps you meant `NodeConfiguration`? ...which no longer has those constants.

Reply:
Changing port numbers, as for this test they simply need to be unused ports. It might make the intent of the test clearer too.